pegasus/migrations/80_add_purpose_and_region_columns_to_hoc_survey_prizes.rb

```diff
@@ -0,0 +1,19 @@
+Sequel.migration do
+  up do
+    alter_table(:hoc_survey_prizes) do
+      drop_index :claimant
```

Comment:
how big is this table? it would be more performant to add the new index first so that it can use the old index to create it.